src/components/relationships.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { AsyncMultiSelect, InlineFormLabel, Switch } from '@grafana/ui';
+import { get, set } from 'lodash';
+import CogniteDatasource from '../datasource';
+import { SelectedProps } from './queryEditor';
+
+const MultiSelectAsync = (props) => {
+  const { datasource, query, onQueryChange, selector, placeholder } = props;
+  const { refId } = query;
+  const s = selector.rout.split('.');
+  return (
+    <AsyncMultiSelect
+      loadOptions={() => datasource.getRelationshipsDropdowns(refId, selector)}
+      value={[...get(query, s)]}
+      defaultOptions
+      allowCustomValue
+      onChange={(values) => onQueryChange(set(query, s, values))}
+      placeholder={placeholder}
+      maxMenuHeight={150}
+    />
+  );
+};
+export const Relationships = (
```

Comment:
```suggestion
export const RelationshipsTab = (
```

and the filename as well